instana-java-sdk/src/main/java/com/instana/sdk/support/SpanSupport.java

```diff
@@ -261,27 +324,35 @@ public static void inheritNext(String traceId, String spanId) {
   }
 
   /**
-   * Clears an open span, i.e. ends a currently open span without committing it.
+   * Clears the currently open SDK span, i.e. ends the currently open span without committing it.
+   *
+   * @since 1.2.0
+   */
+  public static boolean clearCurrent() {
+    return false;
+  }
+
+  /**
+   * Clears an open SDK span of type {@code type}, i.e. ends a currently open span without committing it.
    *
    * @param type
    *          The type of span to clear.
-   * @deprecated Use {@link SpanSupport#clearCurrent(Span.Type, String)}.
    */
-  @Deprecated
-  public static void clearCurrent(Span.Type type) {
-    /* empty */
+  public static boolean clearCurrent(Span.Type type) {
+    return false;
   }
 
   /**
-   * Clears an open span, i.e. ends a currently open span without committing it.
+   * Clears an open SDK span of type {@code type} with name {@code name},
```

Comment:
- We never specify what an SDK span is
- It is not clear this method does not affect AutoTrace spans
- What do we clear if there are multiple spans with same type and name on the stack? The last?
- If type is `ENTRY`, and the name matches the currently entry span, _assuming_ it is created via the SDK, to we clear also all the children?